<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mercurial::Internal::MercurialCommitPanel</class>
 <widget class="QWidget" name="Mercurial::Internal::MercurialCommitPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>229</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="infoGroup">
     <property name="title">
      <string>General Information</string>
     </property>
     <layout class="QFormLayout" name="infoFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="repositoryLabelLabel">
        <property name="text">
         <string>Repository:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="repositoryLabel">
        <property name="text">
         <string>repository</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="branchLabelLabel">
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="branchLabel">
        <property name="text">
         <string>branch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="editGroup">
     <property name="title">
      <string>Commit Information</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="authorLabel">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="authorLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="emailLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>161</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
